pkg/orchestrator/util.go

```diff
@@ -84,9 +86,11 @@ func (n NodeType) String() string {
 		return "ReplicaSet"
 	case K8sService:
 		return "Service"
+	case K8sStatefulSet:
+		return "StatefulSet"
 	default:
-		log.Errorf("Trying to convert unknown NodeType iota: %v", n)
-		return ""
+		log.Errorf("Trying to convert unknown NodeType iota: %d", n)
+		return "Unknown"
```

Comment:
maybe we should return `%d` here to avoid sharing the cache if multiple types are missing (should not happen though)

Reply:
You mean returning a stringified `%d`? Yes, we can do that. It is unlikely but I don't see why not 👍